src/components/rules-list-page/SyncRulesPage.jsx

```diff
@@ -67,20 +69,34 @@ class SyncRulesPage extends React.Component {
                     .map(e => e.name)
                     .join(", "),
         },
-        { name: "frequency", text: i18n.t("Frequency"), sortable: true },
+        {
+            name: "frequency",
+            text: i18n.t("Frequency"),
+            sortable: true,
+            getValue: ({ frequency }) =>
+                isValidCronExpression(frequency)
```

Comment:
Same than before, does it make sense to have `syncRule.isValidFrequency()` or something like that?

Reply:
See https://github.com/EyeSeeTea/metadata-synchronization/pull/134/commits/d4eebaf7a6ac2897e547c59cf1986b291d022208